cli.py

```diff
@@ -15,7 +15,7 @@
 @click.option('--resume_verify', is_flag=True, help="Resume verification of the existing OSF static mirror, need to "
                                                     "import task file")
 @click.option('--delete', is_flag=True, help="Delete nodes from the mirror that have been deleted by users. Requires "
-                                            "compile_active-produced active-node taskfile")
+                                             "'compile_active-produced active-node taskfile' ")
```

Comment:
Why quotes? 

Console:
`Delete nodes from the mirror that have been deleted by users. Requires 'compile_active-produced active-node taskfile'`